### ISSUE TITLE
[Talent] Frost mage, Fingers of Frost stacks

### ISSUE THREAD

**Issue by tvshadowz** [WoWManiaUK/Redemption#1463]
Talent - Fingers of Frost - http://www.wowhead.com/spell=112965/fingers-of-frost
Spell - Ice Lance - http://www.wowhead.com/spell=30455/ice-lance

What should happen: When ice lanced is used with a stack of fingers of frost present, the damage of such should be increased. 

What is currently happening: The first ice lance used whilst having two stacks of fingers of frost has increased damaged however the second ice lance (fingers of frost still present) does not have increased damage. 

See image.

![testr](https://user-images.githubusercontent.com/30256554/29041503-fa2ffff8-7baa-11e7-944b-92851b0ff6cf.png)

**Comment by hotdoggidy** [comment]
This may not be the most appropriate suggestion but perhaps its the LAST stack that doesnt work properly. Might be wishful thinking but maybe adding another stack of the buff could solve the issue? 

FoF gives 3 stacks but only works for the first 2 kind of thing. Solving the bug by adding one? again, maybe not appropriate but just an idea.

**Comment by tvshadowz** [comment]
Hi there all!

After further investigation here is some more information:

"his buff does not stack damage against a frozen target, it simply pretends the target is frozen, meaning it will make no difference against an already frozen target." - [wow-head comments](http://www.wowhead.com/spell=112965/fingers-of-frost#comments)

"Just to clarify the tooltip for anyone perhaps with wishful thinking and who also noticed the grammatical ambiguity: Once you do your "boosted"  Ice Lance you will receive no further benefit for a single charge of Fingers of Frost. In other words the 25% bonus Ice Lance damage does not apply to repeated applications of Ice Lance throughout the 15 seconds for that single charge of FoF." - Elagen

So the first ice lance will act as the target is frozen but the second one wont as the target isn't frozen on that hit. Slightly complicated mechanic but it is working as intended. 

Mages should aim to use ice lance in the first hit for the increased damage and then deep freeze afterwards. 

Thanks all!

**Comment by hotdoggidy** [comment]
Currently on Red it is impossible to Deep freeze after using your first charge to ice lance the target as the 2nd charge DOES NOT treat the target as frozen. "...it simply pretends the target is frozen..." This is the problem i have been trying to address. The first charge of FoF IS treating the target as frozen, hence the extra damage on first ice lance and the ability to cast deep freeze. The SECOND charge however, is NOT treating anything as frozen, NO bonus damage on second ice lance and NO deep freeze.

All the evidence put forth on this issue has been misinterpreted information. 

The first quote you have posted above refers to someone who has frozen a target and expected FoF ice lances to multiply damage based on FoF treating targets as frozen. Ex: "I freeze a target with frost nova, then get FoF charges and ice lance the target, it should treat the target as frozen twice" This is false and i agree.

The second quote u have above refers to a SINGLE charge of FoF. Players cannot use 1 charge to enhance multiple ice lances during the duration of the charge. Ex: "I have 1 charge of FoF left but im 35 yards from the target. if i cast fast enough i can cast 2 ice lances and they should land before the buff expires, allowing both to deal bonus damage." This is also false and i agree. 

Neither of these examples is dealing with what we have happening on Red.

Let me try this again. Fingers of Frost gives "2" charges. "Each" charge is consumed when the player hits the target with a spell. "Each" charge should treat the target as frozen regardless of current state of the target (ie. stunned, frozen, immobilized). This is NOT happening on Red. If u need proof of this specific issue go to a training dummy and try the following:

1. stand very still.
2. cast Frostbolts 1 at a time (do not chain)
3. when FoF procs read the buff.
4. Use 1 charge by casting a single ice lance.
5. Try to use the 2nd charge to cast Deep Freeze.
6. Cry.
7. Re-open this issue until a solution is found.

**Comment by hotdoggidy** [comment]
Im not trying to sound like a smart-ass. i know i dont have all the info or all the answers but the issue frost mages deal with on Mania are basic RTFC mechanics. Tooltip clearly says 2 charges and the spells are NOT responding to the buff properly. All it takes is 1 hour of actually playing a frost mage at 80 to realize there is something terribly wrong with FoF. 

Ice lance alone works fine. Deep Freeze alone works fine. Its the FoF charges that are misbehaving. The 2nd one clocks in for work but gets nothing done. Leaving all the work up to the 1st charge. 

Give me a dev for 10 minutes and i can show exactly what all this mess is about.

**Comment by hotdoggidy** [comment]
Related info i have found: https://www.youtube.com/watch?v=oUeeQKY4zz4

This mage procs FoF but only gets 1 charge. The charge is **NOT** consumed by their frostbolt. They use the charge for an ice-lance that follows a Frostbolt. The difference here is the number of charges and the fact frostbolt is not consuming charges.

If we are only meant to get 1 charge i can see where all the confusion lies. The issue then becomes the wording of the talent/spell, the number of charges and the consumption of said charges when casting certain spells.

**Comment by hotdoggidy** [comment]
i guess we arent discussing this here. Ill start a new issue then. FoF is still broken. All evidence put forth by contributors has been nonsense about specific things not a part of the issue at hand.
